xarray/plot/plot.py

```diff
@@ -193,11 +199,26 @@ def line(darray, *args, **kwargs):
     aspect = kwargs.pop('aspect', None)
     size = kwargs.pop('size', None)
     ax = kwargs.pop('ax', None)
+    hue = kwargs.pop('hue', None)
+    x = kwargs.pop('x', None)
+    add_legend = kwargs.pop('add_legend', True)
 
     ax = get_axis(figsize, size, aspect, ax)
 
-    xlabel, = darray.dims
-    x = darray.coords[xlabel]
+    if ndims == 1:
+        xlabel, = darray.dims
+        if x is not None and xlabel != x:
+            raise ValueError('Input does not have specified dimension ' + x)
```

Comment:
please use string formatting instead so this still works even if the dimensions is not a string, e.g.,
```
raise ValueError('Input does not have specified dimension {!r}'.format(x))
```
(Using non-strings as dimension names is rare but we still try to support it when feasible.)